client/web/compose/src/components/PageBlocks/ChartConfigurator.vue

```diff
@@ -32,6 +32,7 @@
 
     <template v-if="isDrillDownAvailable">
       <b-form-group
```

Comment:
If no chart selected, the open chart button redirects to chart list
Change the tooltip accordingly